dotCMS/src/main/java/com/dotcms/system/event/local/type/staticpublish/AllStaticPublishEndpointsSuccessEvent.java

```diff
@@ -0,0 +1,33 @@
+package com.dotcms.system.event.local.type.staticpublish;
+
+import com.dotcms.publishing.PublisherConfig;
+import com.dotcms.system.event.local.type.publish.PublishEvent;
+
+import java.util.Date;
+
+/**
+ * Object used to represent an event to be triggered when static bundles were sent successfully to all
+ * endpoints
+ *
+ * @author Oscar Arrieta.
+ */
+public class AllStaticPublishEndpointsSuccessEvent extends PublishEvent {
+
+    private PublisherConfig config = null;
+
+    public AllStaticPublishEndpointsSuccessEvent(PublisherConfig config) {
+        setName(AllStaticPublishEndpointsSuccessEvent.class.getCanonicalName());
+        setPublishQueueElements(config.getAssets());
+        setDate(new Date());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use the Java 8 Date and Time API instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2143)